lib/datadog/tracing/sampling/span/ext.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Datadog
+  module Tracing
+    module Sampling
+      module Span
+        # Checks if a span conforms to a matching criteria.
+        class Ext
```

Comment:
* 1️⃣ Should this be `module Ext`?
* 2️⃣ This description seems kinda weird for a `module Ext` ;)